server/src/main/java/org/opensearch/cluster/routing/allocation/allocator/BalancedShardsAllocator.java

```diff
@@ -347,8 +415,9 @@ static class WeightFunction {
         private final float theta1;
         private AllocationConstraints constraints;
         private RebalanceConstraints rebalanceConstraints;
+        private final float preferPrimaryBalanceBuffer;
```

Comment:
This is only being used in the ctor. Lets remove this if you are not planning to use this anywhere else.

Reply:
Ack.